cf-oscore/src/main/java/org/eclipse/californium/oscore/OptionJuggle.java

```diff
@@ -328,4 +329,73 @@ public static OptionSet merge(OptionSet eOptions, OptionSet uOptions) {
 		return eOptions;
 	}
 
+	/**
+	 * Retrieve RID value from an OSCORE option.
+	 * 
+	 * @param oscoreOption the OSCORE option
+	 * @return the RID value
+	 */
+	static byte[] getRid(byte[] oscoreOption) {
+		if (oscoreOption.length == 0) {
+			oscoreOption = new byte[] { 0x00 };
```

Comment:
For me this case will always return `null`, or?

Reply:
Changed in commit 9a654a3f67270da32a728120f62cfc0e5919d242